crates/hyperdrive-math/src/long/close.rs

```diff
@@ -53,10 +53,54 @@ impl State {
 
         Ok(flat + curve)
     }
+
+    /// Calculates the market value in shares of a long position using the equation:
+    /// market_estimate = trading_proceeds - flat_fees_paid - curve_fees_paid
+    ///
+    /// trading_proceeds = flat_value + curve_value
+    ///                  = dy * (1 - t) + dy * t * p
```

Comment:
You can surround in-line math with a back-tick followed by a dollar-sign ( `` `$` dy * (1-t) `$` `` ) to have it rendered with latex, like elsewhere in the [docs](https://docs.rs/hyperdrive-math/latest/hyperdrive_math/struct.State.html#method.open_long_curve_fee)